.github/workflows/update_blackhole.yml

```
@@ -0,0 +1,30 @@
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Runs every day at 2:00 AM UTC
+  workflow_dispatch:  # Allows manual triggering
+
+jobs:
+  update-blackhole-file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)